services/course.js

```diff
@@ -0,0 +1,28 @@
+const Course = require('~/models/course')
+const uploadService = require('~/services/upload')
+const { ATTACHMENT } = require('~/consts/upload')
+
+const courseService = {
+  createCourse: async (author, data) => {
+    const { title, description, lessons, attachments } = data
+    let attachmentUrls
```

Comment:
maybe it would be better to set value of empty array y default if there are no attachments passed

Reply:
i don't really see any pros of it cause it still be an empty array as the result